backend/worker.py

```
@@ -0,0 +1,46 @@
+from dotenv import load_dotenv
+
+load_dotenv()
+
+# Reads the URL from your local .env file (which is not pushed to GitHub)
+AMQP_URL = os.getenv("AMQP_URL")
+
+if not AMQP_URL:
+    print("[-] ERROR: Missing AMQP_URL variable in .env file!")
+    exit(1)
+
+def callback(ch, method, properties, body):
```

Ruff (0.15.15)

[warning] Unused function argument: `ch`

(ARG001)